pages/common/xml-format.md

```diff
@@ -0,0 +1,28 @@
+# xml format
+
+> Format an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Format an XML document, indenting with tabs:
+
+`xml format --indent-tab {{path/to/input.xml|URI}} > {{path/to/output.xml}}`
+
+- Format an HTML document, indenting with 4 spaces:
+
+`xml format --html --indent-spaces {{4}} {{path/to/input.html|URI}} > {{path/to/output.html}}`
+
+- Recover parsable parts of a malformed XML document, without indenting:
```

Comment:
I think it reads better without a comma, but I'm not totally sure.
```suggestion
- Recover parsable parts of a malformed XML document without indenting:
```

Reply:
I've been using a comma where two flags or alternate flags are involved to help the user see the correlation.

Reply:
left as-is.